.github/workflows/website-release-update.yaml

```
@@ -0,0 +1,59 @@
+name: Check and Update Release Version
+
+on:
+  schedule:
```

this would enable to trigger the workflow manually (and not wait for the cronjob).
```suggestion
  workflow_dispatch:
  schedule:
```